src/main/java/pl/touk/ticketBooking/domain/Timetable/TimetableService.java

```diff
@@ -0,0 +1,88 @@
+package pl.touk.ticketBooking.domain.Timetable;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import pl.touk.ticketBooking.domain.Bill.Bill;
+import pl.touk.ticketBooking.domain.Guest.Guest;
+import pl.touk.ticketBooking.domain.Guest.GuestRepository;
+import pl.touk.ticketBooking.domain.Movie.Movie;
+import pl.touk.ticketBooking.domain.Movie.MovieRepository;
+import pl.touk.ticketBooking.domain.Room.Room;
+import pl.touk.ticketBooking.domain.Room.RoomRepository;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Comparator;
+import java.util.List;
+
+import static pl.touk.ticketBooking.domain.DataValidation.ValidateInput.validateNameAndSurname;
+import static pl.touk.ticketBooking.domain.DataValidation.ValidateInput.validateReservationTime;
+import static pl.touk.ticketBooking.domain.Seat.Seat.reservingSeatsRules;
+
+@Service
+@RequiredArgsConstructor
+public class TimetableService {
+
+    private final TimetableRepository timetableRepository;
+    private final GuestRepository guestRepository;
+    private final MovieRepository movieRepository;
+    private final RoomRepository roomRepository;
+
+    public ResponseEntity<List<Timetable>> getTimetable() {
+        return new ResponseEntity<>(timetableRepository.findAll(), HttpStatus.OK);
+    }
+
+    @SneakyThrows
+    public List<Timetable> getRepertoire(String day, String sessionTime) {
+        LocalDate date = LocalDate.parse(day);
+        LocalTime time = LocalTime.parse(sessionTime, DateTimeFormatter.ofPattern("HH:mm"));
+        return timetableRepository.findByDay(date, time);
+    }
+
+    @SneakyThrows
+    public Timetable findById(long id) {
+        return timetableRepository.findById(id).orElseThrow();
+    }
+
+    @SneakyThrows
+    @Transactional
+    public ResponseEntity<Bill> addTickets(Guest guest, long id) {
```

Comment:
Zwróć uwagę, że tak nazwany parametr `long id` niewiele mówi czytelnikowi. Trudno po tym wywnioskować czy to jest id biletu, filmu, gościa czy czegoś jeszcze innego. 

To bardzo ważne w kontekście łatwości rozwijania i utrzymywania kodu w przyszłości.